src/v2/api/controllers/AuthController.ts

```diff
@@ -175,6 +208,12 @@ export class AuthController {
     return this.authService.resetPassword(token, body);
   }
 
+  @ApiOkResponse()
+  @ApiBadRequestResponse({
```

Comment:
You've missed InvalidBodyException in VerificationEmailDTO and TooManyActionsException in requestEmailVerification method

Reply:
Done